diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/spaces/IndentationRuleFixTest.kt

```diff
@@ -230,4 +105,173 @@ class IndentationRuleFixTest : FixTestBase("test/paragraph3/indentation",
+    @TestMethodOrder(MethodOrderer.DisplayName::class)
+    inner class MultilineStringLiterals {
+        @ParameterizedTest(name = "extendedIndent = {0}")
+        @ValueSource(booleans = [false, true])
+        @DisplayName("no whitespace should be injected (code matches settings)")
+        @Tag(WarningNames.WRONG_INDENTATION)
+        fun noWhitespaceShouldBeInjectedCodeMatchesSettings(extendedIndent: Boolean, @TempDir tempDir: Path) {
+            val defaultConfig = IndentationConfig("newlineAtEnd" to false)
+            val customConfig = defaultConfig.withCustomParameters(*extendedIndent(enabled = extendedIndent))
+
+            val whitespaceInStringLiterals = when {
+                extendedIndent -> whitespaceInStringLiteralsContinuationIndent
+                else -> whitespaceInStringLiteralsSingleIndent
+            }
+
+            lintMultipleMethods(
+                whitespaceInStringLiterals,
+                tempDir = tempDir,
+                rulesConfigList = customConfig.asRulesConfigList())
+        }
+
+        @ParameterizedTest(name = "extendedIndent = {0}")
+        @ValueSource(booleans = [false, true])
+        @DisplayName("no whitespace should be injected (mis-indented code reformatted)")
+        @Tag(WarningNames.WRONG_INDENTATION)
+        fun noWhitespaceShouldBeInjectedMisIndentedCodeReformatted(extendedIndent: Boolean, @TempDir tempDir: Path) {
+            val defaultConfig = IndentationConfig("newlineAtEnd" to false)
+            val customConfig = defaultConfig.withCustomParameters(*extendedIndent(enabled = extendedIndent))
+
+            val whitespaceInStringLiteralsActual = when {
+                extendedIndent -> whitespaceInStringLiteralsSingleIndent
+                else -> whitespaceInStringLiteralsContinuationIndent
+            }
+            val whitespaceInStringLiteralsExpected = when {
+                extendedIndent -> whitespaceInStringLiteralsContinuationIndent
+                else -> whitespaceInStringLiteralsSingleIndent
+            }
+
+            lintMultipleMethods(
+                actualContent = whitespaceInStringLiteralsActual,
+                expectedContent = whitespaceInStringLiteralsExpected,
+                tempDir = tempDir,
+                rulesConfigList = customConfig.asRulesConfigList())
+        }
+    }
+
+    /**
+     * See [#1340](https://github.com/saveourtool/diktat/issues/1340).
+     *
+     * @see IndentationRuleWarnTest.ExpressionsWrappedAfterOperator
+     */
+    @Nested
+    @DisplayName("Expressions wrapped after operator")
+    @TestMethodOrder(MethodOrderer.DisplayName::class)
+    inner class ExpressionsWrappedAfterOperator {
+        /**
+         * @see IndentationRuleWarnTest.ExpressionsWrappedAfterOperator.shouldBeProperlyIndented
+         */
+        @ParameterizedTest(name = "extendedIndentAfterOperators = {0}")
+        @ValueSource(booleans = [false, true])
+        @DisplayName("should be properly indented")
```

Comment:
In kotlin you can name the test with `tilda`. This is why it was added to language. Making an annotation hell with `@DisplayName` is a bad practice, to my mind. For me code becoming unreadable

Reply:
Ok, will remove `@DisplayName` wherever possible.